adhocracy4/comments_async/static/comments_async/moderator_statement.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export const ModeratorStatement = ({ last_edit: lastEdit, statement }) => {
+  const translated = {
+    moderator: 'Moderator'
+  }
+  return (
+    <div className="row">
+      <div className="a4-comments__moderator-statement__container">
+        <i className="fas fa-share a4-comments__moderator-statement__icon" />
+        <div className="a4-comments__moderator-statement__content">
+          <div className="a4-comments__moderator-statement__moderator">
+            {translated.moderator}
+          </div>
+          <div className="a4-comments__moderator-statement__text">
+            {lastEdit}
```

Comment:
do we want to get it from backend like we did similarily here?
https://github.com/liqd/adhocracy4/pull/996/commits/c0af4f052ca5e726db97cf4128568a713629e66b

the other way would reintroduce the javascript `localeString` thingy, but I guess we rather want to have once consistent way how we make dates localized and timzoned?

Reply:
Ups, I didn't see your comment. But yes! Let's do it the same way!